{"name": "Проверка авторизации курьера с неправильно указанным логином или паролем", "status": "passed", "description": "Если авторизоваться под несуществующим пользователем, запрос возвращает ошибку", "attachments": [{"name": "stdout", "source": "2fb2e2c1-89e3-4716-bfd7-ca6d20dd93c9-attachment.txt", "type": "text/plain"}], "start": 1721921920444, "stop": 1721921920615, "uuid": "ce442f8c-5c0a-45a3-b424-084a55612b40", "historyId": "1a3b7146c1dfbb02e9b219fcd14ff12a", "testCaseId": "1a3b7146c1dfbb02e9b219fcd14ff12a", "fullName": "tests.test_courier.TestCourierLogin#test_courier_login_wrong_login_pass__error_message", "labels": [{"name": "epic", "value": "test_courier_login"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "DESKTOP-619AU3M"}, {"name": "thread", "value": "8372-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier"}]}